packages/katello/katello/katello.spec

```diff
@@ -194,8 +194,9 @@ Useful utilities for managing Katello services
 %{_sysconfdir}/bash_completion.d/katello-service
 
 %changelog
-* Wed Jul 17 2019 Evgeni Golov - 3.12.1-1
-- Release 3.12.1
+* Sun Jul 21 2019 Jonathon Turel <jturel@gmail.com> 3.12.1-2
```

Comment:
This entry should be above Evgeni's for 3.12.1-1 but other than that I think it's good